vector/src/main/res/values/strings.xml

```diff
@@ -3054,6 +3054,8 @@
     <!-- TODO remove key -->
     <string name="live_location_bottom_sheet_stop_sharing" tools:ignore="UnusedResources">Stop sharing</string>
     <string name="live_location_bottom_sheet_last_updated_at">Updated %1$s ago</string>
+    <string name="live_location_not_enough_permission_dialog_title">You don’t have permission to share locations</string>
+    <string name="live_location_not_enough_permission_dialog_description">You need to have the right permissions in order to share locations in this room.</string>
```

Comment:
Do those two wording mention `live location` instead of `location`? I think this is always possible to share (static) location, so this may be confusing for the user.

Reply:
I agree, replaced.